can_tools/scrapers/official/MT/mt_vaccinations.py

```diff
@@ -18,11 +18,16 @@ class MontanaCountyVaccine(ArcGIS):
             measurement="cumulative",
             unit="people",
         ),
-        "Dose_2": CMU(
+        "Fully_Vaxed": CMU(
```

Comment:
I bet the MT scraper was a bit old, but could use update the values here to use the variable constants?

Reply:
@ghop02 yeah I can add the variables!

I don't know much (anything) about re-running for the old data--but I can look into it. Do you know where/how that might be done?